spec/unit/bricks/middleware/logger_middleware_spec.rb

```diff
@@ -62,4 +64,19 @@
       end
     end
   end
+
+  describe '.call' do
+    tmp_dir = Dir.mktmpdir
+    log_dir = tmp_dir + '/logs'
+    execution_id = 'execid'
+    let(:params) { { 'execution_id' => execution_id, 'log_directory' => log_dir } }
+
+    it 'set MSF compatible file logger' do
+      log_file = "#{log_dir}/#{execution_id}.log"
```

Comment:
warning: Lint/UselessAssignment: Useless assignment to variable - `log_file`. (rubocop)